src/minerva/services/merge_processors.py

```
@@ -0,0 +1,553 @@
+
+        return final_content, merge_history
+
+
+class HeadingMergeProcessor(MergeProcessor):
+    """
+    Heading-based merge strategy.
+
+    Groups content by markdown headings across all notes.
+    """
+
+    def process_merge(
```

Pylint (3.3.7)

[refactor] Too many local variables (18/15)

(R0914)